br/cmd/br/search.go

```diff
@@ -1,6 +1,6 @@
 // Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
 
-package restore
+package main
```

Comment:
It is somehow strange that the `search` command is directly in the main package, maybe create a new package for it?

Reply:
done